src/webgpu/shader/execution/expression/call/builtin/subgroupBroadcast.spec.ts

```diff
@@ -76,139 +115,58 @@ g.test('data_types')
     t.selectDeviceOrSkipTestCase(features);
   })
   .fn(async t => {
+    const wgSize = [4, 1, 1];
```

Comment:
Are we losing meaningful coverage by limiting to this?

I think no because the workgroup_uniform_load test checks over kWGSizes.

I think this is ok.  But noted for your consideration

Reply:
The `compute,all_active`, `compute,split`, and `broadcastFirst,split` also iterate over many WG sizes. The fragment tests iterate over many framebuffer sizes.

My thinking is the data types don't need such a thorough cross product. Across the various tests I think enough coverage exists.